apps/qa/src/pages/edde/helpers.py

```diff
@@ -3,12 +3,15 @@
 import re
 import streamlit as st
 
+from dcpy.configuration import PUBLISHING_BUCKET
 from dcpy.utils import s3
 from dcpy.connectors import github
 from dcpy.connectors.edm import publishing
 
 REPO_NAME = "db-equitable-development-tool"
 PRODUCT = "db-eddt"
+assert PUBLISHING_BUCKET, "PUBLISHING_BUCKET must be defined"
```

Comment:
(nit/idea) would doing `os.environ["PUBLISHING_BUCKET"]` instead of `os.environ.get("PUBLISHING_BUCKET")` in `configuration.py` effectively be the same thing as all the downstream assert statments?

I often can't decide between environ[] and environ.get() and maybe `mypy` is a significant factor here

Reply:
My thought was that we only need to throw an error if we're actually trying to use that part of dcpy - we could have a job which calls our geocoding utilities (which totally exist) with cli that doesn't need to interact with s3 buckets, so it's fine if those env vars aren't set

Reply:
There's probably a slightly more elegant approach than what I have done